roles/test_snmpTraps/tasks/main.yml

```diff
@@ -15,12 +15,6 @@
       for i in {1..3}; do oc delete po -l application=default-interconnect;  done
   changed_when: false
 
-- name: "Check for snmpTraps logs"
```

Comment:
Make sure to double check that these SNMP tests still pass after this change, since you're decreasing the wait time by ~77% here.

The command itself actually looks suspect to me - I would expect to put a sleep in there to standardize the wait length, otherwise isn't it basically dictated by the speed of the API response? The examples from our "STF Feature Verification" wiki use `sleep 1`.

Reply:
Done

Reply:
I'm still skeptical of this test and don't think it will be reliable after this change (but it could have been reliable before the PR since it ran 10 extra times)

The default alertmanager route we describe in the docs requires 30s of disruption at minimum before it will send an alert. We don't want to sleep for long because the pod may be able to restart and transmit some data, so we should keep the `sleep 1` but increase the loop size to guarantee that this lasts at least 30s. I have proposed what I think is the best solution (though it's going to run much longer than even the old code before this PR)